### DISCUSSION TITLE
Invalid Code compared to Google Authenticator

### DISCUSSION THREAD

**Discussion by bmelika** [Authenticator-Extension/Authenticator#711] [General]
Hi there,

I have a very weird issue..I have 3 accounts transferred from Google Authenticator to Chrome Authenticator..2 are working perfectly and the 3rd is always not in sync with Google. I tried to sync time million times but never had the same code..I tried to generate 100 times a QR code from Google Authenticator but the same happens every time, any suggestions?

**Comment by Sneezry** [comment]
Hi @bmelika , we have noticed we had a bug in migration from Google authenticator app for some accounts. A fix has already been commited and will be avaliable in the next release.

See https://github.com/Authenticator-Extension/Authenticator/issues/701
